Classes/Controller/StandardController.php

```diff
@@ -36,7 +36,7 @@ public function indexAction()
         $this->view->assign('flowPathWeb', realpath(FLOW_PATH_WEB));
         $this->view->assign('isPackageAvailable', $this->packageManager->isPackageAvailable('MyCompany.MyPackage'));
 
-        $baseUri = $this->request->getHttpRequest()->getAttribute(ServerRequestAttributes::BASE_URI);
+        $baseUri = (string)RequestInformationHelper::generateBaseUri($this->request->getHttpRequest());
         $this->view->assign('baseUri', (string)$baseUri);
```

Comment:
Since it is cast to string above already…
```suggestion
        $this->view->assign('baseUri', $baseUri);
```